src/tests/data/dummy.py

```diff
@@ -0,0 +1,20 @@
+def main(app_config=None, q1=0, q2=2):
+    q1 = q1 * 3
+    if q1 > q2: # pylint: disable=R1705 no-else-return
```

Comment:
Why not delete the line with else: and instead just return if q1<= q2 to prevent the pylint. The pylint seems a bit odd here